### DISCUSSION TITLE
Custom syntax for smaller font

### DISCUSSION THREAD

**Discussion by ottovw** [remarkjs/remark#1077] [Q&A] [answered]
Hi,

we're trying to achieve to have 2 font-sizes when using the react-markdown and thought of  using `<`  as the syntax:

```
Here's my normal font
< The smaller font
```

the goal is tolook something like this:

![image](https://user-images.githubusercontent.com/1045946/204136383-2a019e8c-a83f-4215-8112-a2afa9d5bbd9.png)

Does something like this exist already (wasn't able to google it)  or do you have some pointer for a tutorial on how to write a respective plugin? Had troubles finding a good entry point.

Thanks,
Otto

**Comment by ChristianMurphy** [comment, marked as answer]
https://github.com/micromark/micromark#extending-markdown

Offers advise.
The general recommendation is don't create yet another flavor of markdown if you don't need to.

https://github.com/remarkjs/remark-directive would likely better suit most people's need, potentially including yours, as it is semi-standard.

If you do choose to take on creating new syntax anyway. https://github.com/micromark/micromark#creating-a-micromark-extension introduces how to create a syntax extension.

**Reply by ottovw** [reply]
@ChristianMurphy thanks, great help! remark-directive seems like the goto solution. Will try to implement and once successful, accept your answer.
